documentation/docs/reference/services/Event.md

```diff
@@ -404,13 +401,14 @@ Response format:
 }
 ```
 
-
 POST /event/checkin/
 ----------------------------
 
 Retrieves a struct that contains information about the event checkin status, point increment value, and total point number.
 Takes in a struct that contains an event checkin code.
 
+Valid values for `status` are `Success`, `InvalidTime`, `AlreadyCheckedIn`. When `status != Success`, the `newPoints` and `totalPoints` fields will be -1 and should be ignored.
```

Comment:
Is there anything stopping us from removing newPoints and totalPoints from the struct? I don't think any other endpoint uses them.

Reply:
I thought it might be helpful for the frontend for updating information, but if they're already making calls to the event, it isn't as necessary (like re-rendering the profile and showing an updated point amount).

Reply:
> Looks good to me, is the DeleteProfile endpoint going to be disabled for the whole event?

Yeah, I think it might be risky to have it available. I don't know if any front ends are using it at all. @PatrickKan